.github/workflows/build-docs.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          ref: ${{ github.event.pull_request.head.ref }}
+          ref: ${{ github.sha }}
```

Comment:
`ref: ${{ github.event.pull_request.head.sha }}` is what was used in the previous docs workflow. Please use that instead.

Reply:
I have updated the ref.